astroquery/eso/core.py

```diff
@@ -26,9 +27,30 @@
 from ..exceptions import RemoteServiceError, NoResultsWarning, LoginError
 from ..query import QueryWithLogin
 from ..utils import schema
+import pyvo
 
 __doctest_skip__ = ['EsoClass.*']
 
+def decorator_with_params(dec):
```

Comment:
There are deprecation decorators in astropy, is there a particular reason you choose not to use those?

Reply:
Thanks for the info, I am now using them.